app/src/main/java/com/rahobbs/todo/activities/TodoPagerActivity.java

```diff
@@ -0,0 +1,68 @@
+package com.rahobbs.todo.activities;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+
+import com.rahobbs.todo.R;
+import com.rahobbs.todo.fragments.TodoFragment;
+import com.rahobbs.todo.helpers.TodoItem;
+import com.rahobbs.todo.helpers.TodoLab;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/5/16.
+ * Activity for ViewPager of to-do item details
+ */
+public class TodoPagerActivity extends AppCompatActivity{
```

Comment:
total nitpick, spacing after the word `AppCompatActivity` before the `{`

as you're writing if you hit `option` + `cmd` + `L` the code will auto format for you and fix a lot of these little formatting issues. Also if you totally loose your mind down the road you can write your own style guide and when you auto format Android Studio will apply your styles rules. 

I can send you a view style guides if you want :+1: